bin/ch/ch.cpp

```diff
@@ -671,6 +671,10 @@ HRESULT ExecuteTest(const char* fileName)
         }
     }
 Error:
+    if( fileContents != nullptr)
+    {
+        free((char*)fileContents);
```

Comment:
> Actually, virtually no instance of ExternalArrayBuffer in ch actually cleans up its memory properly

What do you mean exactly? Current runtime stores a reference to the ExternalArrayBuffer when it's used as source, and when GC finally collects the ExternalArrayBuffer instance it'll clean up.